bindings/python/src/boost_python.hpp

```diff
@@ -7,6 +7,8 @@
 
 #include <cstdio>
 #include <libtorrent/aux_/disable_warnings_push.hpp>
+// https://github.com/boostorg/system/issues/32#issuecomment-462912013
+#define HAVE_SNPRINTF
```

Comment:
I would be more comfortable with this being conditioned by an old version of `_MSC_VER`